.bumpversion.cfg

```diff
@@ -0,0 +1,15 @@
+[bumpversion]
```

Comment:
Not sure about this due to our use of conda, but this can be replaced with [setuptools_scm](https://pypi.org/project/setuptools-scm/).

Reply:
Last time i tried setuptools_scm (ages ago), I don't think I got a working conda recipe. I tend to like bumpversion. But yeah, one more step to version, than setuptools_scm, and potential for errors if not using bumpversion to commit.

Reply:
okay, I got that working. It does not feel intrgrated with conda at all (had to use it in a hacky way). But at least there's only a single source of truth now :)